server/sonar-server/src/main/java/org/sonar/server/plugins/ws/PluginWSCommons.java

```diff
@@ -142,9 +143,44 @@ public void writeArtifact(JsonWriter jsonWriter, Release release) {
     jsonWriter.endObject();
   }
 
+  /**
+   * Write an "update" object to the specified jsonwriter.
+   * <pre>
+   * "update": {
```

Comment:
this method writes the `update` (no **s**) object and so documentation is in sync

did your comment actually apply to another method?